### DISCUSSION TITLE
OpenDTU Fusion v2.1 - OpenDTU-OnBattery - victron smart shunt

### DISCUSSION THREAD

**Discussion by akihilananda** [hoylabs/OpenDTU-OnBattery#1092] [General]
Guten Tag.
OpenDTU-OnBattery
pinmapping.json Einstellungen:
"battery": {
            "rx": 13,
            "tx": 14
        },
        "victron": {
            "rx": 44,
            "tx": 43,
            "rx2": -1,
            "tx2": -1

Hardware- Einstellungen:
13, 14 aktiv
43, 44 aktiv
Problem:
In der Live Ansicht werden keine Daten abgefragt.
Ich bitte um Hilfe

**Comment by schlimmchen** [comment]
Hast du im Web-Interface die Batterieschnittstelle akiv? Und hast du "VE.Direct" im Web-Interface eingeschaltet? Was steht im Log?

**Reply by akihilananda** [reply]
Hallo vielen Dank für ihre Antwort, inzwischen habe ich das Problem gelöst.

Meine Konfiguration:
openDTUOnbattery V 2024.01.26
openDTU Fusion v2.1 mit OpenDTU Fusion CAN/Iso Shield
ve direkt Kabel (Nach Github Anleitung)
Display sh1106
Victron smart shunt
Victron mppt 100 20
Hoymiles HMS 500

Es funktioniert alles einwandfrei.
Respekt an die Entwickler.
Endlich kann ich die Nulleinspeisung über Raspberry und Python Script vergessen.
Jetzt werden alle Daten von smart shunt und mppt 100 20 angezeigt.
Ohne Fusion CAN/Iso Shield gehts halt nicht.
pinmapping.json wurde angepasst nachdem ich Pin Belegung von opendtu Fusion gelesen habe 
"battery": {
            "rx": 11,
            "tx": -1    
        },
        "victron": {
            "rx": 10,
            "tx": -1,
            "rx2": -1,
            "tx2": -1

Vielen Dank noch mal für ihre Rückmeldung.
